storage/statedb/proof_test.go

```diff
@@ -105,6 +770,26 @@ func mutateByte(b []byte) {
 	}
 }
 
+func increseKey(key []byte) []byte {
+	for i := len(key) - 1; i >= 0; i-- {
+		key[i]++
+		if key[i] != 0x0 {
+			break
+		}
+	}
+	return key
+}
+
+func decreseKey(key []byte) []byte {
+	for i := len(key) - 1; i >= 0; i-- {
+		key[i]--
+		if key[i] != 0xff {
+			break
+		}
+	}
+	return key
+}
+
```

Comment:
There's benchmark functions for VerifyRange in `go-ethereum/trie/proof_test.go`. What about adding them?
`BenchmarkVerifyRangeNoProofXX`

Reply:
Right, but I will add them later since it doesn't affect the snapshot logic at all.